src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -16,6 +16,26 @@ object ExportVCF {
     case _ => "1"
   }
 
+  def strVCF(element_type: Type, a: Annotation) : String = {
+    element_type match{
```

Comment:
type will never be null.  Also, this should take the StringBuilder as an input so we don't need to allocate